### PR TITLE
style(weave): add icons to trace tree

### PR DESCRIPTION
**Pull request by jwlee64**
## Description

I think this is what we want, mostly copied from @m-rgba pr here
https://github.com/wandb/weave/pull/3218/files#diff-fd135f6c043e8503d22da884b0464e16851a3364b605c6e95894c93aec355fcb

Screenshot 2024-12-12 at 3 13 08 PM

## Testing

How was this PR tested?

### CODE REVIEW

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx
Comment (m-rgba) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx:1]: 👍 Cool. TY!

**Review by m-rgba — APPROVED** [review]
LGTM. TY

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx [hunk 1]
```diff
@@ -43,21 +31,34 @@ export const PlaygroundCallStats = ({call}: {call: TraceCallSchema}) => {
...
+        />
         {(call.output as any)?.choices?.[0]?.finish_reason && (
-          <>
-            <span>
-              Finish reason: {(call.output as any).choices[0].finish_reason}
-            </span>
-            <span>•</span>
-          </>
+          <Tooltip
+            content="Finish Reason"
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx:49]: Prefer sentence case

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx [hunk 2]
```diff
@@ -43,21 +31,34 @@ export const PlaygroundCallStats = ({call}: {call: TraceCallSchema}) => {
...
+            trigger={
+              <span>
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx:51]: I think a Pill renders a div, so this should probably be a block level element like div too.

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx [hunk 3]
```diff
@@ -43,21 +31,34 @@ export const PlaygroundCallStats = ({call}: {call: TraceCallSchema}) => {
...
+                <Pill
+                  icon="checkmark-circle"
+                  label={(call.output as any).choices[0].finish_reason}
+                  color="moon"
+                  className="-ml-[8px] bg-transparent text-moon-500 dark:bg-transparent dark:text-moon-500"
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundChat/PlaygroundCallStats.tsx:56]: I didn't know you could write the negative margin that way in Tailwind, I think I've always done it `ml-[-8px]`. Interesting!

**Review by jamie-rasmussen — APPROVED** [review]

**Pull request merged**